docs/swagger.yaml

```diff
@@ -204,96 +204,183 @@ components:
       x-examples:
         example-1:
           imageId: laptop_image4433
+    ImageId:
```

Comment:
why name it ID when the only property is ID?

Reply:
i havent made changes here, i think it was a stoplight bug. I'll try to figure out what this should be 😅

Reply:
aha - there arent any actually changes here, but git is confused since i moved it around in the code. Guess its called imageId since there already is a object called image.

We could however remove the object

Reply:
removed